components/index/SiteTopUpper/StayingPopulation.vue

```diff
@@ -3,7 +3,7 @@
     <div class="StayingPopulation">
       <div class="StayingPopulation-title">
         <h3 class="StayingPopulation-heading">
-          {{ $t('都内の滞在人口の増減状況') }}
+          {{ $t('都内の滞在人口の増減状況（毎週月曜日更新）') }}
```

Comment:
```suggestion
          {{ $t('都内の滞在人口の増減状況')}}{{ $t('（毎週月曜日更新）') }}
```
と分割しておいた方が，更新のタイミングが変わる場合に「都内の滞在人口の増減状況」が翻訳されない事態を防げるのでよいと思います．

Reply:
フォローありがとうございます！これなのですが、翻訳時に文脈がわかるようになるべくセンテンスは分けずにおいています（特に英訳で、意味が通じない、テキストの体裁が合わない、などの不具合が生じたため）。ここの場合は、判断は別れるところではあると思うのですが、現在の運用上（古い翻訳キーを削除しない）、当面は翻訳されない事態は起きないと思われるので、こちらのPRは変更せずにいきたいと思います。

Reply:
@kaizumaki 承知しました．ありがとうございます．